src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,17 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  status: '',
+};
+
+const categoriesSlice = createSlice({
+  name: 'category',
+  initialState,
+  reducers: {
+    checkstatus: (state) => ({ ...state, status: 'Under Construction' }),
```

Comment:
- kindly, update the `categories` state to under construction to fit the requirement 👍 
```javascript
checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
```